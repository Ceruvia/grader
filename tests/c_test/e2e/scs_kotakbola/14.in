199 669 261
B 24
H 75
H 78
E
