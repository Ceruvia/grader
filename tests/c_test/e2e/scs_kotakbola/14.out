153 0 24
Ya
